edx_exams/apps/core/api.py

```diff
@@ -99,8 +99,11 @@ def update_attempt_status(attempt_id, to_status):
         if not allowed_to_start:
             raise ExamIllegalStatusTransition(error_msg)
 
-        attempt_obj.start_time = datetime.now(pytz.UTC)
-        attempt_obj.allowed_time_limit_mins = _calculate_allowed_mins(attempt_obj.user, attempt_obj.exam)
+        # Once start time, and end time by extension, has been set further transitions to started
+        # must not update this value
+        if not attempt_obj.start_time:
```

Comment:
the only path for restart we support right now is a full reset.  That gives the user an entirely new exam attempt rather than restarting one that is already in progress.